RELEASE_NOTES.md

```diff
@@ -1,3 +1,12 @@
+### Version 2.2.6 (2017)
+
+* Add Tomcat 8.5.x alongside 8.0.x
+* Updated tomcat versions to latest release version
+* Requires tomcat configuration in build.gradle:
```

Comment:
Are you referring to httpProtocol and ajpProtocol? These are the parameters that worked for me. It would seem natural to make them the defaults. I don't know Tomcat or the plugin well enough in order to comment on this.